event-handlers/src/test/java/no/sikt/nva/nvi/events/evaluator/PointCalculationTest.java

```diff
@@ -0,0 +1,627 @@
+            .withNorwegianCreatorAffiliatedWith(nviOrganization1.hasPart())
+            .getExpandedPublication();
+
+    var candidate = getEvaluatedCandidate(publication);
+    var actualContributorPoints =
+        candidate.institutionPoints().getFirst().creatorAffiliationPoints();
+    assertThat(actualContributorPoints)
+        .usingRecursiveFieldByFieldElementComparatorIgnoringFields("nviCreator")
+        .containsExactlyInAnyOrderElementsOf(expectedPoints);
+  }
+
+  private List<CreatorAffiliationPoints>
+      getExpectedPointsForSingleContributorWithTwoAffiliations() {
+    var creatorPoint1 =
+        new CreatorAffiliationPoints(
+            null, nviOrganization1.hasPart().get(0).id(), asBigDecimal("0.5"));
+    var creatorPoint2 =
+        new CreatorAffiliationPoints(
+            null, nviOrganization1.hasPart().get(1).id(), asBigDecimal("0.5"));
+    return List.of(creatorPoint1, creatorPoint2);
+  }
+
+  @Test
+  void shouldCalculatePointsForCreatorAffiliations() {
+    var expectedPoints = getExpectedPointsForTwoContributorsWithThreeAffiliations();
+    var publication =
+        factory
+            .withTopLevelOrganizations(nviOrganization1, nviOrganization2)
+            .withNorwegianCreatorAffiliatedWith(nviOrganization1.hasPart())
+            .withNorwegianCreatorAffiliatedWith(nviOrganization1.hasPart().getFirst())
+            .getExpandedPublication();
+
+    var candidate = getEvaluatedCandidate(publication);
+    var actualPoints = candidate.institutionPoints().getFirst().creatorAffiliationPoints();
+    assertThat(actualPoints)
+        .usingRecursiveFieldByFieldElementComparatorIgnoringFields("nviCreator")
+        .containsExactlyInAnyOrderElementsOf(expectedPoints);
+  }
+
+  private List<CreatorAffiliationPoints>
+      getExpectedPointsForTwoContributorsWithThreeAffiliations() {
+    var creatorPoint1 =
+        new CreatorAffiliationPoints(
+            null, nviOrganization1.hasPart().get(0).id(), asBigDecimal("0.25"));
+    var creatorPoint2 =
+        new CreatorAffiliationPoints(
+            null, nviOrganization1.hasPart().get(1).id(), asBigDecimal("0.25"));
+    var creatorPoint3 =
+        new CreatorAffiliationPoints(
+            null, nviOrganization1.hasPart().get(0).id(), asBigDecimal("0.5"));
+    return List.of(creatorPoint1, creatorPoint2, creatorPoint3);
+  }
+
+  private static Stream<PointParameters>
+      twoCreatorsAffiliatedWithTwoDifferentInstitutionsPointProvider() {
+    return Stream.of(
+        // example from cristin calculations, publicationYear 2022
+        new PointParameters(
+            "AcademicArticle",
+            "Journal",
+            "LevelOne",
+            false,
+            2,
+            asBigDecimal("0.7071"),
+            asBigDecimal("0.7071"),
+            asBigDecimal("1.4142")));
+  }
+
+  private static Stream<PointParameters> twoCreatorsAffiliatedWithOneInstitutionPointProvider() {
+    return Stream.of(
```

Comment:
In general, I return `Stream<Named<Something>>`, which is as easy as adding `Named.of("Words saying what I am inputting", new PointsParameters(...))` and `@ParameterizedTest(name = "The test name")`, which makes all our lives easier after the fact because it links the setup with the expectations and prints it nicely in the test report